src/omotes_sdk/omotes_interface.py

```diff
@@ -184,6 +194,10 @@ def connect_to_submitted_job(
         :param auto_disconnect_on_result: Remove/disconnect from all queues pertaining to this job
             once the result is received and handled without exceptions through
             `callback_on_finished`.
+        :param auto_dead_letter_after_ttl: When erroneous situations occur (e.g. client is offline),
+            the job result message (if available) will be dead lettered after the given TTL,
+            and all queues of this job will be removed subsequently. Default to 48 hours if unset.
+            Set to `None` to turn off auto dead letter and clean up.
```

Comment:
```suggestion
            Set to `None` to turn off auto dead letter and clean up but be aware this may lead to messages and queues to be stored in RabbitMQ indefinitely (which use up memory & disk space).
```

Reply:
Updated in 3691c6257cb2daa7130fb39e33bb8f64de43e7ab